intercode/envs/swe/swe_env.py

```diff
@@ -37,7 +37,7 @@ def reset_container(self) -> None:
             user = "ziyuewang" if "ZiyueWang25" in repo_name else "swe-bench"
             clone_cmd = f"git clone https://github.com/{user}/{repo_name}.git"
             self.logger.debug(f"Clone: {clone_cmd}")
-            is_valid = self.exec_action(clone_cmd)
```

Comment:
why do we use self.container.exec_run instead of self.exec_action? is it because we don't want to be constraint by the timeout?

Reply:
I switched back to use self.exec_action with 3 minutes timeout. It works fine at this moment! 
We'd better use self.exec_action because it records execution information in self.info for later tracking and it returns "is_valid" properly.

Reply:
Yeah, that sounds reasonable to me! I switched to exec_run to avoid the timeout, didn't consider the information recording.